### PR TITLE
Basic Login Page

### PR DESCRIPTION
**Pull request by Da-Scher**
This PR adds a Login webpage that can switch between a register form and a login form by selecting the bottom-most span (either 'Already have an account?' or 'Need an account?' depending on the mode.) The page defaults to login mode.

### CODE REVIEW

File: client/src/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  console.log(registerMode);
+  return (
+    <div>
+      <div> { registerMode ?
+        (<div>
+          <h1>Register</h1>
+            <form onSubmit={handleSubmit}>
+            <div>
+              <label htmlFor="username" >Email</label>
```
Comment (justinsprecco) [inline client/src/Login.jsx]:
```suggestion
              <label htmlFor="username" >Username</label>
```

Comment (justinsprecco) [reply client/src/Login.jsx]: Everything else looks great, but this one adjustment.

**Review by justinsprecco — CHANGES_REQUESTED** [review]
Nice work!

File: client/src/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            <div>
+              <label htmlFor="password" >Password</label>
+              <input type="password" id="password" value={password} onChange={(e) => setPassword(e.target.value)}/>
+            </div>
+            <button type="submit">Register Now</button>
+            </form>
+        </div>) : (<div><h1>Login</h1>
+            <form onSubmit={handleSubmit}>
+            <div>
+              <label htmlFor="username" >Email</label>
```
Comment (justinsprecco) [inline client/src/Login.jsx]:
```suggestion
              <label htmlFor="username" >Username</label>
```

File: client/src/Login.jsx (hunk 1 above)
Comment (Da-Scher) [reply client/src/Login.jsx]: Fixed in commit [7c9ca79](https://github.com/justinsprecco/Acumen/pull/4/commits/7c9ca79ec5db0a6c795fae729d5549b1d2e7ff16)

File: client/src/Login.jsx (hunk 2 above)
Comment (Da-Scher) [reply client/src/Login.jsx]: Fixed in commit [7c9ca79](https://github.com/justinsprecco/Acumen/pull/4/commits/7c9ca79ec5db0a6c795fae729d5549b1d2e7ff16)

**Review by justinsprecco — APPROVED** [review]

**Pull request merged**